src/Microsoft.ML.HalLearners/VectorWhitening.cs

```diff
@@ -464,7 +464,7 @@ private static void GetColTypesAndIndex(IHostEnvironment env, IDataView inputDat
             }
         }
 
-        public override void Save(ModelSaveContext ctx)
+        private protected override void SaveModel(ModelSaveContext ctx)
```

Comment:
>SaveModel [](start = 40, length = 9)

It's good that we're finally locking this down, `protected` members like these are a little appreciated part of our public API we need to lock down. Thanks @artidoro!